### ISSUE TITLE
Unable to build multus cni image

### ISSUE THREAD

**Issue by cgchinmay** [k8snetworkplumbingwg/multus-cni#693]
Tried building the image but getting following error
```
docker build -t chinmay5j/multus .

#8 330.9 Building plugins
#8 923.5 go: github.com/k8snetworkplumbingwg/network-attachment-definition-client@v1.1.1-0.20210510153419-66a699ae3b05 requires
#8 923.5 	k8s.io/kube-openapi@v0.0.0-20200410145947-61e04a5be9a6 requires
#8 923.5 	github.com/NYTimes/gziphandler@v0.0.0-20170623195520-56545f4a5d46: invalid version: git fetch --unshallow -f origin in /root/go/pkg/mod/cache/vcs/749e556e4213b8d2c698b43bceeb37764e9d6e8bd7ad455d3b7dc3f8a88b4c59: exit status 128:
#8 923.5 	fatal: git fetch-pack: expected shallow list
#8 923.5 	fatal: The remote end hung up unexpectedly
------
executor failed running [/bin/sh -c rpm --import https://mirror.go-repo.io/centos/RPM-GPG-KEY-GO-REPO &&     curl -s https://mirror.go-repo.io/centos/go-repo.repo | tee /etc/yum.repos.d/go-repo.repo &&     yum install -y $INSTALL_PKGS &&     rpm -V $INSTALL_PKGS &&     cd /usr/src/multus-cni &&     ./hack/build-go.sh]: exit code: 1
```

Dockerfile that I used
```
# This Dockerfile is used to build the image available on DockerHub
FROM centos:centos7 as build

ENV GOPROXY direct

# Add everything
ADD . /usr/src/multus-cni

ENV INSTALL_PKGS "git golang-1.13.10-0.el7.x86_64"
RUN rpm --import https://mirror.go-repo.io/centos/RPM-GPG-KEY-GO-REPO && \
    curl -s https://mirror.go-repo.io/centos/go-repo.repo | tee /etc/yum.repos.d/go-repo.repo && \
    yum install -y $INSTALL_PKGS && \
    rpm -V $INSTALL_PKGS && \
    cd /usr/src/multus-cni && \
    ./hack/build-go.sh

FROM centos:centos7
LABEL org.opencontainers.image.source https://github.com/k8snetworkplumbingwg/multus-cni
COPY --from=build /usr/src/multus-cni /usr/src/multus-cni
WORKDIR /

ADD ./images/entrypoint.sh /
ENTRYPOINT ["/entrypoint.sh"]
```

Go and git versions
```
go version go1.15.6 darwin/amd64

git version 2.29.2
```

**Comment by dougbtv** [comment]
Fixed by #695

**Comment by dougbtv** [comment]
Thank you @cgchinmay for the issue and the fix!!
